.github/workflows/ci.yml

```diff
@@ -116,6 +116,7 @@ jobs:
           fi
           echo SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }} >> $GITHUB_ENV;
           echo FRONTEND_SENTRY_DSN=${{ secrets.FRONTEND_SENTRY_DSN }} >> $GITHUB_ENV;
+          echo BACKEND_SENTRY_DSN=${{ secrets.BACKEND_SENTRY_DSN }} >> $GITHUB_ENV;
           echo SENTRY_ORG=${{ secrets.SENTRY_ORG }} >> $GITHUB_ENV;
           echo SENTRY_PROJECT=${{ secrets.SENTRY_PROJECT }} >> $GITHUB_ENV;
           echo SENTRY_URL=${{ secrets.SENTRY_URL }} >> $GITHUB_ENV;
```

Comment:
Given that these are no longer needed, can we have them removed?

Reply:
these are needed by the sentry webpack plugin. will rename them to clarify that they are for the frontend sentry project.

Reply:
If I'm not mistaken, `SENTRY_URL` defaults to https://sentry.io?

Reply:
just remembered why i didn't rename them earlier. because sentry automatically looks for the env vars and adds to them to the config. i'll add them to config explicitly so it's clearer

Reply:
yea looking at the docs again, it seems like most of these envs are optional: https://github.com/getsentry/sentry-webpack-plugin. we just need SENTRY_DSN, SENTRY_AUTH_TOKEN, and environment set through NODE_ENV

Reply:
Specify the env as `LAMBDA_NODE_ENV`. Never figured out why webpack couldn't build when `NODE_ENV` was set to `production`

Reply:
turns out `SENTRY_ORG` and `SENTRY_PROJECT` are required
![image](https://user-images.githubusercontent.com/18087283/112956736-564bb200-9173-11eb-9c95-41a84e49b33b.png)